backend/utils/llm.py

```diff
@@ -178,7 +178,8 @@ def get_memory_summary(uid: str, memories: List[Memory]) -> str:
     Remember {user_name} is busy so this has to be very efficient and concise.
     Respond in at most 50 words.
   
-    Output your response in plain text, without markdown.
+    Output your response in plain text, without markdown. No newline character and only use numbers for the action items.
+    The total words in the response should not exceed 50 words.
```

Comment:
we have the limit at 179, should we remind llm about the max numbers of words?

Reply:
It's upto us, I added at the end again to enforce the limit. Should I remove it?

Reply:
keep the prompt simple as much as posible, try removing it and see what llm give us man.

Reply:
Done! Gives the same response